src/Italy/Mews.Fiscalizations.Italy/XmlSigner.cs

```diff
@@ -6,9 +6,9 @@ namespace Mews.Fiscalizations.Italy
 {
     public static class XmlSigner
     {
-        public static XmlDocument Sign(XmlDocument document, X509Certificate2 certificate)
+        public static XmlDocument Sign(XmlElement element, X509Certificate2 certificate)
         {
-            var signedXml = new SignedXml(document)
+            var signedXml = new SignedXml(element)
```

Comment:
Isn't this actually a breaking change? Are we sure it works?

Reply:
I think it should be fine as the constructor takes accepts either the document or the element (and from the element u can get the document).

But i cannot confirm, will need to test it. (i hope we have some test credentials for that :D)

Reply:
we're not using this endpoint anywhere (we use uniwix only) + we don't have test credentials for the italian fiscalizations

Reply:
Ehm, then let's remove it I guess.